pdf_gui.py

```diff
@@ -87,15 +94,20 @@ def open_file():
             total_pages = pdf_file.num_pages
             images = pdf_file.page_images
             current_page = 1
-            ### TO-DO: Revert the comment below to code after implementing the display_page function. ###
-            ## Comment: The code below gives a hint on what needs to be included as arguments for the display_page function.
-            # display_page(images[current_page - 1])
-            ### END OF TO-DO. ###
-            
+            self.display_page(current_page - 1, pdf_file)
+            self.update_page_label()
+
     # Displays a single page to the GUI.
     # Updates page label and zoom based on whether the zoom buttons and previous page or next_page button is used.
-    def display_page():
-        pass
+    def display_page(self, page_number, pdf_file):
```

Comment:
Taking in pdf_file as an argument may be unnecessary since the pdf_file is a global variable and information regarding the PDF file such as the images and total page numbers is already saved upon opening the PDF file.

However, the page_number argument is fine because if we were to implement a jump_to_page function (where the user is able to type in the page number they want to go to) in the future, adding that as an argument for that function will be neccesary.